code/game/objects/items/tcg/tcg.dm

```diff
@@ -0,0 +1,297 @@
+
+GLOBAL_LIST_EMPTY(cached_guar_rarity)
+GLOBAL_LIST_EMPTY(cached_rarity_table)
+//Global list of all cards by series, with cards cached by rarity to make those lookups faster
+GLOBAL_LIST_EMPTY(cached_cards)
+
+/obj/item/tcgcard
+	name = "Coder"
+	desc = "Wow, a mint condition coder card! Better tell the Github all about this!"
+	icon = 'icons/obj/tcg.dmi'
+	icon_state = "runtime"
+	w_class = WEIGHT_CLASS_TINY
+	var/id = -1 //Unique ID, for use in lookups and storage, used to index the global datum list where the rest of the card's info is stored
+	var/series = "coderbus" //Used along with the id for lookup
+	var/flipped = FALSE
+
+/obj/item/tcgcard/Initialize(mapload, brand, badge_me)
```

Comment:
You are using `series`, `template`, `brand` and `badge_me` for var names.
Could you use more explictly clear ones?
Stuff like `template_datum`, `template_id` or the likes?

Reply:
sure, that's no trouble